features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -178,8 +178,9 @@
         <!--<AccessTokenValueGenerator>org.wso2.carbon.identity.oauth.tokenvaluegenerator.SHA256Generator</AccessTokenValueGenerator>-->
 
         <!-- This configuration is used to specify whether the Service Provider tenant domain should be used when generating
-             access token. Otherwise user domain will be used. Currently this value is only supported by the JWTTokenIssuer.-->
-        <!--<UseSPTenantDomain>True</UseSPTenantDomain>-->
+             access token. Otherwise user domain will be used. Currently this value is only supported by the JWTTokenIssuer.
```

Comment:
```suggestion
             access token. Otherwise user's tenant domain
 will be used. Currently this value is only supported by the JWTTokenIssuer.
```